unity/Assets/Plugins/Countly/CountlyManager.cs

```diff
@@ -195,9 +196,15 @@ protected void EndSession(long duration)
 
       builder.Append("&session_duration=");
       AppendConnectionData(builder, duration.ToString());
+			Log ("Requesting session end");
+		try {
+				WebRequest www = WebRequest.Create(appHost + "/i?" +builder.ToString());
+				www.GetResponse().Close();
 
-      ConnectionQueue.Enqueue(builder.ToString());
```

Comment:
About WebRequest and using it in queue. It does work with NET 2.0 and 2.0 subset, but is not included with micro stripping.